plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -808,15 +810,26 @@ test.describe( 'Add New Variable Product Page', () => {
 			'#variable_product_options .toolbar-top a.expand_all'
 		);
 		await page.check( 'input.checkbox.variable_manage_stock' );
-		await page.fill( 'input#variable_regular_price_0', variationOnePrice );
+
+		const firstVariationContainerLocator = await page
```

Comment:
Definitely not a blocking issue, but a small bit of feedback I'd over is that the constant name probably doesn't need to include "Locator" to be sufficiently descriptive.  `firstVariationContainer` is a great name IMO.

Reply:
fixed: https://github.com/woocommerce/woocommerce/pull/37854/commits/876b7d688cb0ede2cc0df74e92f6f08e644a930f